core/src/main/java/org/apache/accumulo/core/clientImpl/ClientTabletCacheImpl.java

```diff
@@ -660,6 +660,7 @@ private void requestTabletHosting(ClientContext context,
     }
 
     // System tables should always be hosted
+    // TODO should this return for all sys tables?
     if (AccumuloTable.ROOT.tableId() == tableId || AccumuloTable.METADATA.tableId() == tableId) {
       return;
     }
```

Comment:
Commenting here as a reminder to delete this once resolved

Reply:
Created PR #5486

Reply:
Created https://github.com/apache/accumulo/pull/5487

Reply:
Resolved